segtypes/common/data.py

```diff
@@ -231,11 +231,15 @@ def is_valid_jtbl(self, sym: Symbol, bytes) -> bool:
         if not jtbl_func:
             return False
 
+        # A label of a jump table shouldn't point to the start of the function
+        if word == jtbl_func.vram_start:
+            return False
+
         for i in range(4, len(bytes), 4):
             word = int.from_bytes(bytes[i : i + 4], options.get_endianess())
 
             # If the word doesn't contain an address in the current function, this isn't a valid jump table
-            if not jtbl_func.contains_vram(word):
+            if not jtbl_func.contains_vram(word) or word == jtbl_func.vram_start:
```

Comment:
I think the or you added here should be a separate check, since we don't even want to fall into the next condition in that case - just return false